files/build/scripts/pip

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+  
```

Comment:
Is it true `pip` is just an alias of `pip2`?
If yes, we need to hook pip2.

Reply:
Yes, pip2 added.